cmd/root.go

```
@@ -163,15 +180,19 @@ func startKubelet(nodeId string) {
 		log.L.WithError(err).WithField("value", daemonPortEnv).Fatal("Invalid value from KUBELET_PORT in environment")
 	}
 
+	configMap := k8sClient.CoreV1().ConfigMaps(kubeNamespace)
+	store, err := api.NewKeyValueStore(configMap, configMapName)
```

ignoring error here?